from data.data import Datos
import custom_exceptions

class DatosPlanta(Datos):
    pass